deployment-bundle-steps.md

```diff
@@ -0,0 +1,135 @@
+# How to setup a project with deployment to S3
+
+1. Create S3 bucket for production / staging (your-bucket-name-production & your-bucket-name-staging). (If using cloudfront, see [optional settings](#optional-settings))
```

Comment:
That's a good point... but I think the project may have differing needs, for example, many projects could share one bucket... how would we deal with that?